### DISCUSSION TITLE
More sounds

### DISCUSSION THREAD

**Discussion by Secret-chest** [linuxmint/discussions#94] [Ideas]
Cinnamon really needs more sounds. Many other desktops have these.
### alerts
* error window appears
* warning window appears
* info window appears
* prompt for password
* window needs attention
* critical notification
* important notification
* notification (exists)
* low priority notification
### system
* change volume (exists)
* lock
* unlock
* log in (exists)
* log out (exists)
* empty rubbish
* caps lock (exists in accessibility)
* caps unlock (exists in accessibility)
### window management
* switch workspaces (exists)
* open windows (exists)
* restore windows
* minimise windows (exists)
* close windows (exists)
* tile windows (exists)
* maximise windows (exists)
* minimise windows (exists)

**Comment by navid-zamani** [comment]
Yes, and distinctive identifiable ones too!

I’m no fan of sound minimalism, or minimalism in general.

The sounds should be useful cues. Not iBland iFeatureless iArbitrary iPiano iNotes.

**Comment by HT-7** [comment]
I have turned off all UI sounds and animations, but anyone who wants them should be able to have them.
